pages/commands.vue

```diff
@@ -1,239 +1,103 @@
+          <ul class="commands-list">
+            <CommandComponent v-for="command in columnOne(commands.meta)" :key="command.name" :command="command" />
+          </ul>
+        </div>
+        <div class="flexbox__item commands-column">
+          <ul class="commands-list">
+            <CommandComponent v-for="command in columnTwo(commands.meta)" :key="command.name" :command="command" />
+          </ul>
+        </div>
+      </div>
     </div>
-  </div>
+  </main>
 </template>
 
-<style lang="scss" scoped>
-@import "../assets/config";
-
-.args {
-  position: absolute;
-  left: 50%;
-  top: 80px;
-  opacity: 0;
-  transform: translate(-50%);
-  width: 100%;
-  background-color: #fff;
-  border-radius: 10px;
-  transition: opacity 0.2s;
-  z-index: 1000;
-
-  &.active {
-    opacity: 1;
-  }
-
-  &__item {
-    position: relative;
-    padding: 10px;
-
-    &:after {
-      content: "";
-      position: absolute;
-      left: 50%;
-      bottom: 0;
-      height: 1px;
-      width: 50%;
-      transform: translate(-50%);
-      background-color: darken($primaryColor, 5);
-    }
-
-    &--title,
-    &--desc {
-      color: darken($primaryColor, 5);
-    }
-
-    &--title {
-      font-size: 20px;
-      font-weight: 600;
-      margin-bottom: 5px;
+<script scoped>
+import CommandComponent from '@/components/command'
+import commands from '@/static/commandsData'
+
+export default {
+  name: 'CommandsPage',
+  components: {
+    CommandComponent
+  },
+  data () {
+    return {
+      commands
     }
-
-    &:last-child {
-      padding-bottom: 15px;
-
-      &:after {
-        content: none;
-      }
+  },
+  methods: {
+    columnOne (commands) {
+      return commands.filter(function (command, index) {
+        return index < Math.round(commands.length / 2)
+      })
+    },
+    columnTwo (commands) {
+      return commands.filter(function (command, index) {
+        return index >= Math.round(commands.length / 2)
+      })
     }
   }
 }
+</script>
 
-h3,
-h3.button {
-  display: inline-block;
-  font-size: 1.5em;
-  padding: 0.4rem 1rem;
-  margin: auto;
-  margin-bottom: 1rem;
-}
-
-li {
-  position: relative;
-
-  &:after {
-    content: "";
-    position: absolute;
-    bottom: -2px;
-    height: 2px;
-    width: 180px;
-    left: 50%;
-    transform: translateX(-50%);
-    background-color: darken($primaryColor, 5);
-  }
-}
-
-h3 {
-  border: 1px solid transparent;
-  font-family: "Raleway", sans-serif;
-}
-
-h3.button {
-  cursor: pointer;
-  border: 1px solid $primaryColor;
-  border-radius: 0.3rem;
-  background-color: darken($primaryColor, 5);
-  transition: all 0.15s ease-in-out;
-
-  &:before {
-    content: "↓";
-  }
-
-  &.active,
-  &:hover {
-    background-color: lighten($primaryColor, 5);
-    color: $tertiaryColor;
-    cursor: pointer;
-  }
-}
+<style lang="scss" scoped>
+@import '~@/assets/scss/functions';
+@import '~@/assets/scss/colors';
+@import '~@/assets/scss/settings';
```

Comment:
I think there is a way to load these for every page via Nuxt in nuxt.config.js, without having to import them in every file, since they are general styles

Reply:
I will investigate further, unless you can save me some time by providing a link to some documentation for it, thank you